GoogleForCreators/web-stories-wp
# Apply page template: create new instead of replacing

## Summary
Changes the logic for applying templates.
Instead of replacing the previous current page, a new page is added and selected instead.


## Relevant Technical Choices



## To-do




## User-facing changes
No confirmation is asked now for applying a page template.


## Testing Instructions
- Switch to Page templates tab in the library
- Apply a page template
- Verify that a new page is added right away with the template and a snackbar messages displayed. The new page is selected.


### QA
Check the instructions above.





This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #7063
